indico/modules/events/util.py

```diff
@@ -552,10 +552,14 @@ def register_location_change(entry):
 
 
 def serialize_event_for_ical(event):
+    title = event.title
+    if label := getattr(event, 'label', None):
+        title += f' [{label.title}]'
```

Comment:
```suggestion
    if event.label:
        title += f' [{event.label.title}]'
```

In the other place from which you probably copied the code, `entry` is not necessarily an `Event`. But here it is, so no need for `getattr`.

Reply:
Ah yes, my bad.